index.js

```diff
@@ -52,6 +52,22 @@ function isOneOf (v, values) {
   return values.indexOf(v) !== -1;
 }
 
+function forOwn (object, iterator) {
+  for (var property in object) {
```

Comment:
Just a style thing, but I usually use Object.keys and then iterating over the keys that way.

Reply:
Okay, I can do that. I wasn't sure if you were aiming for maximum compatibility (for example, I didn't see any uses of `Array#forEach`), so I figured I would avoid `Object.keys()`. I'll update it.

Reply:
I just tend to use for loops because they're way faster according to many benchmarks. Probably premature optimization in most cases, but still nice to be consistent with code style if possible